client/components/theme-tier/theme-tier-badge/theme-tier-community-badge.js

```diff
@@ -45,15 +41,28 @@ export default function ThemeTierCommunityBadge() {
 
 	return (
 		<>
-			<ThemeTierBadgeTracker />
+			{ ! legacyCanUseTheme && (
```

Comment:
[Non-blocking] Maybe we could rename `legacyCanUseTheme` to `canUseTheme` I find the naming a bit confusing given that it's not a legacy use case.